### ISSUE TITLE
docker image on hub.docker.com is 10 month old

### ISSUE THREAD

**Issue by aol-nnov** [danger/danger-js#923]
https://hub.docker.com/r/dangersystems/danger-js

Am I looking in the wrong place or it is not automatically built any more?

**Comment by orta** [comment]
Yeah, they got hacked and leaked my GitHub creds, so I turned off my account. I don't use it, so I wasn't going to keep up risky infra for little gain.

**Comment by aol-nnov** [comment]
Sad to hear that..

Well, I've reviewed which grants dockerhub requests from github, and permissions are very broad: full access to public and private repos, issues, etc..

So, for now, it seems to me, that keeping separate account for docker automated builds is sufficient. But it is, ahem, inconvenient!..

No docker builds for now, then.. Sad...

**Comment by aol-nnov** [comment]
Anyway, may be, it's time to give them a second chance? Everyone gets hacked sooner or later nowadays..

**Comment by rohit-smpx** [comment]
We could publish to Github instead https://github.com/features/package-registry ?

**Comment by orta** [comment]
Great point, I have access to that

**Comment by orta** [comment]
Still working on this https://github.com/danger/danger-js/actions
